Bottom: 963c3e7b68d4a759f76c58eaa780d3484637a4de
Top:    f58aa69c3adee5cef59558b360220f1b21d5fdfb
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 22:06:31 -0500

Update getInfo block

---

diff --git a/syntax.php b/syntax.php
index b9afe58..10dca49 100644
--- a/syntax.php
+++ b/syntax.php
@@ -14,12 +14,12 @@ class syntax_plugin_tablecalc extends DokuWiki_Syntax_Plugin {
      */
     function getInfo() {
         return array(
-            'author' => 'Gryaznov Sergey',
-            'email' => 'stalker@narezka.org',
-            'date' => '09-02-17',
+            'author' => 'Christopher Voltz',
+            'email' => 'cjunk@voltz.ws',
+            'date' => '2017-05-01',
             'name' => 'Table Calculations Plugin',
-            'desc' => 'Enables Excel style formulas in table syntax',
-            'url' => 'https://narezka.org/tablecalc',
+            'desc' => 'Adds limited spreadsheet functionality to tables',
+            'url' => 'https://github.com/cvoltz/tablecalc.git'
         );
     }
     function getType() {
